official/resnet/resnet.py

```diff
@@ -279,7 +379,7 @@ class Model(object):
   def __init__(self, resnet_size, num_classes, num_filters, kernel_size,
                conv_stride, first_pool_size, first_pool_stride,
                second_pool_size, second_pool_stride, block_fn, block_sizes,
-               block_strides, final_size, data_format=None):
+               block_strides, final_size, version, data_format=None):
```

Comment:
Version may as well take a default value here of _DEFAULT_VERSION

Reply:
Done. I made the version non-private so cifar10 and imagenet classes could share the default.